src/components/HelpTopics/EnableHelpTopicContext.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { HelpTopicContext } from '@patternfly/quickstarts';
+import { useChromeHelpTopicsApi } from './help-topics';
+
+type EnableHelpTopicContextType = {
+  addTopicName: (topicName: string) => void;
+  deleteTopicName: (topicName: string) => void;
+};
+
+const EnableHelpTopicContext = React.createContext<EnableHelpTopicContextType>({
+  addTopicName: () => {},
+  deleteTopicName: () => {},
```

Comment:
Maybe we can move `useQuickstartCloseOnUnmount` inside this context, so it knows to close sidebar only when the helpTopics list is empty.